java/java-guestbook/README.md

```diff
@@ -50,20 +62,22 @@ For details on how to use this sample as a template in Cloud Code, read the docu
 ### Run the app locally with minikube
 
 #### Edit run configuration
-1. Click the configuration dropdown on the top taskbar and select 'Edit Configurations'.
+1. Click the configuration dropdown in the top taskbar and then click **Edit Configurations**.
 ![image](./img/edit-configurations.png)
 
-2. In the Run/Debug Configurations dialog, select 'Develop on Kubernetes'. This configuration watches for changes, then uses [skaffold](https://skaffold.dev/docs/) to rebuild and rerun your app.
+   The **Develop on Kubernetes** configuration watches for changes, then uses [skaffold](https://skaffold.dev/docs/) to rebuild and rerun your app. You can customize your deployment by making changes to this run configuration or by creating a new Cloud Code: Kubernetes run configuration.
+
 
 3. Under **Run > Deployment**, select 'Deploy locally to a minikube cluster'.
 ![image](./img/run-debug-dialog.png)
 
-4. Click 'OK' to save your configuration. 
+4. Click **OK** to save your configuration. 
 
 
 #### Run the app on minikube
-1. Select 'Develop on Kubernetes' from the configuration drop down and click the run icon. Cloud Code runs your app in a local [minikube](ttps://minikube.sigs.k8s.io/docs/start/) cluster.  
-![image](./img/deploy-config.png)
+1. Select 'Develop on Kubernetes' from the configuration dropdown and click the run icon. Cloud Code runs your app in a local [minikube](ttps://minikube.sigs.k8s.io/docs/start/) cluster.  
+![image](./img/edit-configurations.png)
+
```

Comment:
Another single quotes being replaced with bolding missed here

Reply:
Thanks for catching these! Done